doc/UsersGuide/BuildingRunningTesting/FIRE.rst

```diff
@@ -0,0 +1,250 @@
+
+The biggest difference between the UFS FIRE capability and other modes of the UFS SRW is that a special build flag is required to build the coupled fire behavior code, as described in the instructions below. Aside from that, additional input files, and some fire-specific config settings, building and running is the same as any other use of SRW.
+
+
+.. note::
+
+   Although this chapter is the primary documentation resource for running the UFS FIRE configuration, users may need to refer to :numref:`Chapter %s <BuildSRW>` and :numref:`Chapter %s <RunSRW>` for additional information on building and running the SRW App, respectively. 
+
+Quick Start Guide (UFS FIRE)
+=====================================
+
+Download the Code
+-------------------
+
+Clone the |branch| branch of the authoritative SRW App repository:
+
+.. code-block:: console
+
+   git clone -b develop https://github.com/ufs-community/ufs-srweather-app
+   cd ufs-srweather-app
+
+Checkout Externals
+---------------------
+
+Users must run the ``checkout_externals`` script to collect (or "check out") the individual components of the SRW App from their respective GitHub repositories.
+
+.. code-block:: console
+
+   ./manage_externals/checkout_externals
+
+Build the SRW App with Fire Behavior enabled
+--------------------------------------------
+
+To build the SRW with fire behavior code, use the following command:
+
+.. code-block:: console
+
+   ./devbuild.sh -p=<machine> -a=ATMF
+
+where ``<machine>`` is ``hera``, ``derecho``, or any other Tier 1 platform. The ``-a`` argument indicates the configuration/version of the application to build; in this case, the atmosphere-fire coupling (ATMF).
+
+If UFS FIRE builds correctly, users should see the standard executables listed in :numref:`Table %s <ExecDescription>`. There are no additional files expected, since the CFBM is coupled to the UFS weather model via the same ``ufs_model`` executable.
+
+Load the |wflow_env| Environment
+--------------------------------------------
+
+Load the python environment for the workflow:
+
+.. code-block:: console
+
+   module use /path/to/ufs-srweather-app/modulefiles
+   module load wflow_<machine>
+
+where ``<machine>`` is ``hera``, ``derecho``, or any other Tier 1 platform. 
+
+If the console outputs a message, the user should run the commands specified in the message. For example, if the output says: 
+
+.. code-block:: console
+
+   Please do the following to activate conda:
+       > conda activate srw_app
+
+then the user should run |activate|. Otherwise, the user can continue with configuring the workflow. 
+
+.. _FIREConfig:
+
+Configure Experiment
+---------------------------
+
+Users will need to configure their experiment by setting parameters in the ``config.yaml`` file. To start, users can copy an example experiment setting into ``config.yaml``:
+
+.. code-block:: console
+
+   cd ush
+   cp config.fire.yaml config.yaml 
```

Comment:
This is no longer necessary, right?

Reply:
I'd like to keep this as-is; it's a good idea to not work on top of the example file since that is versioned in the repository.